.github/workflows/pr.yaml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   shared-operator-workflow:
     name: shared-operator-workflow
-    uses: redhat-cop/github-workflows-operators/.github/workflows/pr-operator.yml@v1.0.3
+    uses: trevorbox/github-workflows-operators/.github/workflows/pr-operator.yml@default-helmchart-version
```

Comment:
this is probably not correct, do you need me to create a new tag on github-workflows-operators?

Reply:
its correct now.